makezip.sh

```diff
@@ -6,8 +6,9 @@ TMP_DIR=`mktemp -d`
 
 pushd $TMP_DIR
 pip install -r $CURRENT_DIR/requirements.txt -t .
-cp -ra $CURRENT_DIR/app/. .
+cp -ra $CURRENT_DIR/app/* .
 cp -rp $CURRENT_DIR/app/.env.production .env
+find . -name \*.pyc -delete
```

Comment:
Valid point but the pyc files get generated in my local machine while developing. Currently I'm building the zip from my dev dir which has pyc files. That said maybe it makes sense to have PYTHONDONTWRITEBYTECODE in my local env globally anyway.